django_postmark_utils/models.py

```diff
@@ -1,3 +1,203 @@
 from django.db import models
+from django.utils.functional import lazy
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
 
-# Create your models here.
+
+mark_safe_lazy = lazy(mark_safe, str)
+
+
+class Message(models.Model):
+    """
+    Email message content.
+    """
+
+    message_obj = models.BinaryField(
+        _('Message object'),
+        help_text=_('Serialisation of the originally-sent '
+                    '"email.message.Message" (or a subclass) object, in '
+                    'pickle format')
+    )
+    resend_for_id = models.CharField(
```

Comment:
I'm really not sure about the name `resend_for_id`. How about `message_id` for this one, and `email_id` instead of `msg_id` on `Email`.